docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+
+services:
+  mongo-db:
+    container_name: mongo-db
+    image: mongo:3.6
```

Comment:
Maybe we can push this mongo `4.0` or `4.2` to work with the latest major version of mongo?